new/LIT6368PsaltComm.xml

```diff
@@ -0,0 +1,121 @@
+                <editor role="generalEditor" key="AB"/>
+                <editor key="DN"/>
+                <funder>Akademie der Wissenschaften in Hamburg</funder>
+            </titleStmt>
+            <publicationStmt>
+                <authority>Hiob-Ludolf-Zentrum für Äthiopistik</authority>
+                <publisher>Die Schriftkultur des christlichen Äthiopiens und Eritreas: Eine multimediale
+                            Forschungsumgebung / Beta maṣāḥǝft</publisher>
+                <pubPlace>Hamburg</pubPlace>
+                <availability>
+                    <licence target="http://creativecommons.org/licenses/by-sa/4.0/"> This file is
+                                licensed under the Creative Commons Attribution-ShareAlike 4.0. </licence>
+                </availability>
+            </publicationStmt>
+            <sourceDesc>
+                <p/>
+            </sourceDesc>
+        </fileDesc>
+        <encodingDesc>
+            <p>A digital born TEI file</p>
+        </encodingDesc>
+        <profileDesc>
+            <creation/>
+            <abstract>
+                <p>A Psalter containing <ref type="work" corresp="LIT2000Mazmur">Psalms of David</ref>,
+              <ref type="work" corresp="LIT1828Mahale">Canticles of the Prophets</ref> and
+              <ref type="work" corresp="LIT2362Songof">Song of Songs</ref>
+               that are unified with the <ref type="work" corresp="LIT5986CommPsalter">
+            Amharic commentary</ref> and copied as continuous text.</p>
+            </abstract>
+            <textClass>
+                <keywords>
+                    <term key="AmharicLiterature"/>
+                    <term key="Bible"/>
+                    <term key="ChristianLiterature"/>
+                    <term key="Commentary"/>
+                </keywords>
+            </textClass>
+            <langUsage>
+                <language ident="en">English</language>
+                <language ident="gez">Gǝʿǝz</language>
+                <language ident="am">Amharic</language>
+            </langUsage>
+        </profileDesc>
+        <revisionDesc>
+            <change who="DN" when="2021-01-28">Created entity</change>
+        </revisionDesc>
+    </teiHeader>
+    <text>
+        <body>
+            <div type="bibliography">
+                <listRelation>
+                    <relation name="saws:contains" active="LIT6368PsaltComm" passive="LIT5986CommPsalter"/>
+                    <relation name="lawd:hasAttestation" active="LIT6368PsaltComm" passive="BNFabb41"/>
+                </listRelation>
+
+           <div type="edition">
+             <note>The division into text parts follows <ref type="mss" corresp="BNFabb41"/>.</note>
+              <div type="textpart" subtype="incipit" xml:id="Psalms">
+                 <label>Psalms of David with Amharic commentary</label>
+                 <ab>
+                   <l n="1"/>ተግሣጽ፡ ለጻድቃን፡ ወለኃጥአን።
```

Comment:
I cannot open the guidelines now (they don't load), but I think that the verse should be contained in `<l></l>` (as you have done above).

Reply:
ok